DocSum/docker_image_build/build.yaml

```diff
@@ -11,18 +11,57 @@ services:
       context: ../
       dockerfile: ./Dockerfile
     image: ${REGISTRY:-opea}/docsum:${TAG:-latest}
+
   docsum-ui:
     build:
-      context: ../ui
-      dockerfile: ./docker/Dockerfile
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+      context: ../ui/
+      dockerfile: ./docker/Dockerfile.gradio
     extends: docsum
     image: ${REGISTRY:-opea}/docsum-ui:${TAG:-latest}
-  docsum-react-ui:
+
+  whisper:
+    build:
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+      context: GenAIComps
+      dockerfile: comps/asr/whisper/dependency/Dockerfile
+    extends: docsum
+    image: ${REGISTRY:-opea}/whisper:${TAG:-latest}
+
+  multimedia2text:
     build:
-      context: ../ui
-      dockerfile: ./docker/Dockerfile.react
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+      context: GenAIComps
+      dockerfile: comps/dataprep/multimedia2text/Dockerfile
     extends: docsum
-    image: ${REGISTRY:-opea}/docsum-react-ui:${TAG:-latest}
+    image: ${REGISTRY:-opea}/multimedia2text:${TAG:-latest}
```

Comment:
In GenAIComps [related PR](https://github.com/opea-project/GenAIComps/pull/865/files#diff-022171efe111a160f35a88b9872634570129a881f83bdf3780784499884dcf2d), the image is named `dataprep-multimedia2text`, we should have both the image names synced.

Reply:
Thanks @lianhao to point this, let's unify the image name.